src/status_im2/contexts/chat/bottom_sheet_composer/images/style.cljs

```diff
@@ -0,0 +1,24 @@
+(ns status-im2.contexts.chat.bottom-sheet-composer.images.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(def image-container
+  {:padding-top    12
+   :padding-bottom 8
+   :padding-right  12})
+
+(def remove-photo-container
```

Comment:
Does this needs to be a `defn`?

Reply:
@ibrkhalil Why defn? There is no dynamic properties

Reply:
I thought the color would require this variable to be a function, But I was mistaken.
Thanks for the heads up 🙂